integration/README.md

```diff
@@ -0,0 +1,54 @@
+# Integration Tests
+
+This tests core use cases for u-root such as:
+
+- retrieving and kexec'ing a Linux kernel,
+- uinit (user init), and
+- running unit tests requiring root privileges.
+
+## Usage
+
+Run the tests with:
+
+    go test
+
+When the QEMU arch is not amd64, set the `UROOT_TESTARCH` variable. For
+example:
+    UROOT_TESTARCH=arm go test
+
+Currently, only amd64 and arm are supported.
+
+## Steps to get the kernel (bzImage)
+#Step 1: clone the open source linux directory:
+git clone https://github.com/torvalds/linux.git
+
+#Step 2: enter the linux directory and see the directories and files within it
+cd linux
+ls
+
+#Step 3: checkout the latest Linux version 
+git checkout v4.17
+
+#Step 4: preload a .config file. You need virtio_pci and virtio_net 
+wget https://raw.githubusercontent.com/u-root/u-root/master/.circleci/images/test-image-amd64/config_linux4.17_x86_64.txt
```

Comment:
No need to say 'You need' as that might change over time.
You can say
#Step 4: download a config file from u-root

Reply:
Don't they have to move this file to .config?

Reply:
You are right. We made the necessary changes.

Reply:
Yes, since we do download the u-root's .config, we do have to move it to linux directory and then make the kernel using the new .config. We added that to the comments.